rosbag2_py/test/test_transport.py

```diff
@@ -148,6 +148,36 @@ def test_player_api(storage_id):
     ctx.shutdown()
 
 
+@pytest.mark.parametrize('storage_id', TESTED_STORAGE_IDS)
+def test_player_get_starting_time(storage_id):
+    bag_path = str(RESOURCES_PATH / storage_id / 'talker')
+    assert os.path.exists(bag_path), 'Could not find test bag file: ' + bag_path
+
+    storage_options, _ = get_rosbag_options(bag_path, storage_id)
+
+    play_options = rosbag2_py.PlayOptions()
+    play_options.topics_to_filter = ['topic']
+
+    player = rosbag2_py.Player(storage_options, play_options, 'info', 'rosbag2_player_test')
+    starting_time = player.get_starting_time()
+    assert starting_time is not None, 'Expected a valid starting time'
```

Comment:
Could you instead check that the value corresponds to the exact start time value for the `talker` bag? Same for the duration value below

Reply:
I thought about it. But what value to use as an expected value to check?  
We have full coverage with expected value on the CPP layer.